src/firefly/js/drawingLayers/ComputeWebGridData.js

```diff
@@ -0,0 +1,757 @@
+/*
+ * License information at https://github.com/Caltech-IPAC/firefly/blob/master/License.txt
+ * Lijun
+ * 4/14/16
+ */
+
+import { makeWorldPt, makeImagePt,makeImageWorkSpacePt} from '../visualize/Point.js';
+import ShapeDataObj from '../visualize/draw/ShapeDataObj.js';
+import CoordinateSys from '../visualize/CoordSys.js';
+import CoordUtil from '../visualize/CoordUtil.js';
+import numeral from 'numeral';
+import { getDrawLayerParameters} from './WebGrid.js';
+
+const precision3Digit = '0.000';
+
+const RANGE_THRESHOLD = 1.02;
+
+
+const minUserDistance= 0.25;   // user defined max dist. (deg)
+const maxUserDistance= 3.00;   // user defined min dist. (deg)
+
+var useLabels= true;
+var userDefinedDistance = false;
+
+
+/**
+ * This method does the calculation for drawing data array
+ * @param plot - primePlot object
+ * @param cc - the CoordinateSys object
+ * @return a DrawData object
+ */
+export function getData (plot,  cc){
+
```

Comment:
picky- since it is in it's down file which don't you make this function name to something like `makeGridDrawData`